core/filters/price.js

```diff
@@ -34,7 +34,7 @@ export function price (value, storeView) {
 
   const options = { minimumFractionDigits: fractionDigits, maximumFractionDigits: fractionDigits };
 
-  let localePrice = Math.abs(value.toLocaleString(defaultLocale, options));
+  let localePrice = Math.abs(value.toLocaleString(defaultLocale, options)).toFixed(2);
```

Comment:
I think that you should add `.toFixed(2)` also in line number 30. Without this, there is a big chance that the price will have more than two digits after the comma.

Reply:
done